clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -290,19 +289,16 @@ private void doSend(ClientRequest clientRequest, boolean isInternalRequest, long
             // information itself.  It is also the case when discoverBrokerVersions is set to false.
             if (versionInfo == null) {
                 if ((!discoverBrokerVersions) && (log.isTraceEnabled()))
-                    log.trace("No version information found when sending message of type {} to node {}",
-                            clientRequest.apiKey(), nodeId);
+                    log.trace("No version information found when sending message of type {} to node {}. " +
```

Comment:
Is this log message correct? It seems to imply that we looked for the version information and didn't find it. But it seems like the feature is simply disabled due to `discoverBrokerVersions = false`?